docs/content/development/extensions-core/kafka-extraction-namespace.md

```diff
@@ -2,52 +2,34 @@
 layout: doc_page
 ---
 
-# Kafka Namespaced Lookup
+# Kafka Lookups
 
 <div class="note caution">
 Lookups are an <a href="../development/experimental.html">experimental</a> feature.
 </div>
 
 Make sure to [include](../../operations/including-extensions.html) `druid-namespace-lookup` and `druid-kafka-extraction-namespace` as an extension.
 
-Note that this lookup does not employ a `pollPeriod`.
-
-If you need updates to populate as promptly as possible, it is possible to plug into a kafka topic whose key is the old value and message is the desired new value (both in UTF-8).
+If you need updates to populate as promptly as possible, it is possible to plug into a kafka topic whose key is the old value and message is the desired new value (both in UTF-8) as a LookupExtractorFactory.
 
 ```json
 {
   "type":"kafka",
-  "namespace":"testTopic",
-  "kafkaTopic":"testTopic"
+  "kafkaTopic":"testTopic",
+  "kafkaProperties":{"zookeeper.connect","somehost:2181/kafka"}
 }
 ```
 
 |Parameter|Description|Required|Default|
 |---------|-----------|--------|-------|
-|`namespace`|The namespace to define|Yes||
 |`kafkaTopic`|The kafka topic to read the data from|Yes||
-
-## Kafka renames
+|`kafkaProperties`|Kafka consumer properties. At least"zookeeper.connect" must be specified. Only the zookeeper connector is supported|Yes||
 
 The extension `kafka-extraction-namespace` enables reading from a kafka feed which has name/key pairs to allow renaming of dimension values. An example use case would be to rename an ID to a human readable format.
 
-Currently the historical node caches the key/value pairs from the kafka feed in an ephemeral memory mapped DB via MapDB.
-
-## Configuration
-
-The following options are used to define the behavior and should be included wherever the extension is included (all query servicing nodes):
-
-|Property|Description|Default|
-|--------|-----------|-------|
-|`druid.query.rename.kafka.properties`|A json map of kafka consumer properties. See below for special properties.|See below|
-
-The following are the handling for kafka consumer properties in `druid.query.rename.kafka.properties`
+The consumer properties `group.id` and `auto.offset.reset` CANNOT be set in `kafkaProperties` as they are set by the extension as `UUID.randomUUID().toString()` and `smallest` respectively.
```

Comment:
auto.offeset.reset as smallest means "read all the data available in the topic" otherwise two different servers could replay different changelogs.

group.id means every instance is a unique consumer, so they should be accounted for as different consumers.